ui/views_test.py

```diff
@@ -297,13 +304,17 @@ def test_collection_viewset_create(logged_in_apiclient):
     other_user = UserFactory()
     input_data = {
         'owner': other_user.id,
-        'title': 'foo title'
+        'title': 'foo title',
+        'view_lists': [],
+        'admin_lists': []
     }
     assert client.post(url, input_data, format='json').status_code == status.HTTP_403_FORBIDDEN
 
     # or if does not specify the owner id
     input_data = {
-        'title': 'foo title'
+        'title': 'foo title',
+        'view_lists': [],
+        'admin_lists': []
```

Comment:
I'm not really sure about the value of testing that a POST request with a missing id will be forbidden. I would recommend we remove it entirely unless you think it has real value